src/interactions/dragInteraction.ts

```diff
@@ -0,0 +1,134 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Interaction {
+  export class Drag extends AbstractInteraction {
+    private _dragging = false;
+    private _mouseDispatcher: Plottable.Dispatcher.Mouse;
+    private _dragOrigin: Point;
+    private _dragStartCallback: (p: Point) => any;
+    private _dragCallback: (start: Point, end: Point) => any;
+    private _dragEndCallback: (start: Point, end: Point) => any;
+
+    public _anchor(component: Component.AbstractComponent, hitBox: D3.Selection) {
+      super._anchor(component, hitBox);
+      this._mouseDispatcher = Dispatcher.Mouse.getDispatcher(
+                                <SVGElement> (<any> this._componentToListenTo)._element.node()
```

Comment:
If you think that we can use content, can you place a HACKHACK about the anycasting referencing #1732 ?